doc/fileformat.rst

```diff
@@ -0,0 +1,76 @@
+OSI File Format
+----------------
+
+To save multiple serialized OSI messages into one trace file we use the length of each OSI message and save it before the actual OSI message. 
```

Comment:
There should be a list of the possible file formats:
1) *.osi explanation
2) *.txt explanation, deprecated
3) *txth ...